promptsource/templates/aps/dynahate/0.2.3/templates.yaml

```diff
@@ -0,0 +1,89 @@
+dataset: aps/dynahate
+subset: 0.2.3
+templates:
+  04f4e96b-df7f-4c25-873e-1a20f033fed7: !Template
+    answer_choices: noHate ||| hate
+    id: 04f4e96b-df7f-4c25-873e-1a20f033fed7
+    jinja: '{% if label in [0, 1] %}
+
+      {{text}} Is the sentiment the sentence expresses is a {{answer_choices[1]}}
+      speech or {{answer_choices[0]}} speech? ||| {{answer_choices[label]}}
+
+      {% endif %}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: hate_or_noHate_sentiment_analysis
+    reference: ''
+  25a374b9-c3d9-42b8-90e8-38eafba67926: !Template
+    answer_choices: no ||| yes
+    id: 25a374b9-c3d9-42b8-90e8-38eafba67926
+    jinja: "Would you consider following sentence as hateful? yes or no? {{text}}\
+      \ \n|||\n{{answer_choices[label]}}"
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: following sentence
+    reference: ''
+  32ebf212-28fe-407a-b92f-a00d4760e748: !Template
+    answer_choices: noHate ||| hate
+    id: 32ebf212-28fe-407a-b92f-a00d4760e748
+    jinja: 'Is the sentence: "{{text}}" a {{"hate"}} speech or {{"noHate"}} speech?
+
+      |||
+
+      {{answer_choices[label]}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
+      original_task: true
+    name: hate_or_nohate simple
+    reference: ''
+  3941fb66-092a-4456-bb3c-bbba0de39ded: !Template
+    answer_choices: no ||| yes
```

Comment:
Should this be `yes ||| no` instead? Looks like 0 is hateful and 1 is non-hateful.

Reply:
Fixed in the main dataset.

Reply:
This should be fixed as well here

Reply:
This should be fixed here as well